src/components/Generic/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Button.css';
+
+interface ButtonProps { className: string, onClick?: () => void, buttonTitle?: string }
```

Comment:
Can we haven't `buttonTitle`? Why do we need to render button node if we haven't title? Why did u change it from children to this prop?